static/app/components/feedback/useHaveSelectedProjectsSetupFeedback.tsx

```diff
@@ -46,10 +36,3 @@ export function useHaveSelectedProjectsSetupFeedback() {
     fetching,
   };
 }
-
-export function useFeedbackHasSlug() {
```

Comment:
replaced with a call to `useCurrentFeedbackId()` and then a boolean check on the result.